tedana/workflows/tedana.py

```diff
@@ -692,7 +703,17 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
                     "A denoising pipeline for the identification and removal "
                     "of non-BOLD noise from multi-echo fMRI data."
                 ),
-                "CodeURL": "https://github.com/ME-ICA/tedana"
+                "CodeURL": "https://github.com/ME-ICA/tedana",
+                "Node": {
+                    "Name": info_dict["Node"],
+                    "System": info_dict["System"],
+                    "Machine": info_dict["Machine"],
+                    "Processor": info_dict["Processor"],
+                    "Release": info_dict["Release"],
+                    "Version": info_dict["Version"]
+                },
+                "Python": info_dict["Python"],
+                "Command": tedana_command
```

Comment:
When users use the Python function instead of the CLI, perhaps we could have the function call instead? I don't know if that's an easy thing to do though.

Reply:
That would be cool! I guess we could just write a silly function that turns the function call into a string as part of our utils file.